lib/sprockets/uri_tar.rb

```diff
@@ -35,6 +35,21 @@ def compress
       scheme + compressed_path
     end
 
+    def self.fast_compress(uri, root)
+      if uri.include?('://'.freeze)
```

Comment:
We don't need all the freeze calls. We're using the magic comment in the files:

```ruby
# frozen_string_literal: true
```

Reply:
oh, right, missed that